bcgov/HOUS-permit-portal
# Feature/bphh 516/mapping external api

## Description
Add external api to retrieve integration mapping based on template version id


## What type of PR is this? (check all applicable)


## Related Tickets & Documents
https://hous-bssb.atlassian.net/browse/BPHH-1516


## Steps to QA
Prerequisite:
- Enable api for jurisdiction of choice
- Generate an api key for the jurisdiction and note the token
- Ensure there is a published template version, and note it's id
Steps:
- visit `/integrations/api_docs`
- click "Authorize" button and enter the token noted previously
- Open the `"/permit_applications/versions/{version_id}/integration_mapping` request accordion
- Click "Try out", enter the version id noted previously and click "Execute"
- Should get a response with the integration mapping data
![Screenshot 2024-06-17 at 11 43 44 AM](https://github.com/bcgov/HOUS-permit-portal/assets/32022335/a5a4b31e-614f-48cd-aed4-2b5b350ccf7f)
![Screenshot 2024-06-17 at 12 08 21 PM](https://github.com/bcgov/HOUS-permit-portal/assets/32022335/d66f7f04-c43a-4499-bf46-818dda5714eb)
